### PR TITLE
Bugfix/grad accum

### PR DESCRIPTION
**Pull request by zanghz21**
### Description

- Fix gradient accumulation bug
- Fix train logging
- Fix maniskill-ppo-oft config

### Motivation and Context

- Error gradient accumulation value
- Previously only log the rank 0 training metrics. 
- Previous maniskill-ppo-oft miss essential attributes. 

### How has this been tested?

- run embodiment start script

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: rlinf/algorithms/embodiment/grpo_functions.py [hunk 1]
```diff
@@ -265,37 +265,33 @@ def actor_loss_fn(
...
-    pg_losses = -advantages * ratio  # [bs, len]
+    pg_losses = -advantages * ratio
     pg_losses2 = -advantages * torch.clamp(
         ratio, 1.0 - clip_ratio_low, 1.0 + clip_ratio_high
-    )  # [bs, len]
+    )
 
     if loss_mask is not None:
         # Take the maximum of clipped and unclipped losses
         pg_loss = masked_sum(
```
Comment (guozhen1997) [inline rlinf/algorithms/embodiment/grpo_functions.py]: This line looks equivalent to the following calculation. If so, please call the function in utils directly.
`from rlinf.utils.utils import seq_mean_token_mean`
`pg_loss = seq_mean_token_mean(torch.max(pg_losses, pg_losses2), loss_mask)`

File: rlinf/algorithms/embodiment/grpo_functions.py [hunk 2]
```diff
@@ -265,37 +265,33 @@ def actor_loss_fn(
...
         # Take the maximum of clipped and unclipped losses
         pg_loss = torch.max(pg_losses, pg_losses2).mean()  # float
         pg_clipfrac = torch.gt(pg_losses2, pg_losses).float().mean()  # float
 
     # Compile metrics for logging
     metrics_data = {
-        "actor/loss": pg_loss.detach().item(),
-        "actor/pg_loss": pg_loss.detach().item(),
-        "actor/pg_clipfrac": pg_clipfrac.detach().item(),
+        "actor/raw_loss": pg_loss.detach().item(),
```
Comment (guozhen1997) [inline rlinf/algorithms/embodiment/grpo_functions.py:297]: What is the difference between `actor/raw_loss` and `actor/policy_loss`? It is recommended to keep one and rename it to `pg_loss`.

File: rlinf/workers/actor/fsdp_actor_worker.py [hunk 3]
```diff
@@ -342,6 +347,8 @@ def run_training(self):
             append_to_dict(metrics, data)
 
         mean_metric_dict = {key: np.mean(value) for key, value in metrics.items()}
```
Comment (guozhen1997) [inline rlinf/workers/actor/fsdp_actor_worker.py:351]: For variables that need to be logged, you need to convert them to scalars when adding the metrics dictionary, rather than calculating `np.mean` here.

File: rlinf/algorithms/embodiment/grpo_functions.py (hunk 1 above)
Comment (zanghz21) [reply rlinf/algorithms/embodiment/grpo_functions.py]: No, they are different because the data has been shuffled in a different way.

File: rlinf/algorithms/embodiment/grpo_functions.py (hunk 2 above)
Comment (zanghz21) [reply rlinf/algorithms/embodiment/grpo_functions.py:297]: Currently, there is no difference. However, if we add entropy loss, I want it is included in the raw_loss. (like the current practice in ppo_functions.)
